packages/react-styled-core/src/Button/index.js

```diff
@@ -111,7 +73,10 @@ const Button = forwardRef(
           borderRadius={borderRadius}
           data-selected={selected ? 'true' : undefined}
           px={px}
-          css={css}
+          css={[
```

Comment:
Just to take note there is an issue in v10.
https://github.com/emotion-js/emotion/issues/1629#issuecomment-557732213